docs/fundamentals/toc.yml

```diff
@@ -266,139 +266,141 @@ items:
-        href: ../core/tools/dotnet-new-install.md
-      - name: dotnet new uninstall
-        href: ../core/tools/dotnet-new-uninstall.md
-      - name: dotnet new update
-        href: ../core/tools/dotnet-new-update.md
-      - name: .NET default templates
-        href: ../core/tools/dotnet-new-sdk-templates.md
-      - name: Custom templates
-        href: ../core/tools/custom-templates.md
-    - name: dotnet nuget
-      items:
-      - name: dotnet nuget delete
-        href: ../core/tools/dotnet-nuget-delete.md
-      - name: dotnet nuget locals
-        href: ../core/tools/dotnet-nuget-locals.md
-      - name: dotnet nuget push
-        href: ../core/tools/dotnet-nuget-push.md
-      - name: dotnet nuget add source
-        href: ../core/tools/dotnet-nuget-add-source.md
-      - name: dotnet nuget disable source
-        href: ../core/tools/dotnet-nuget-disable-source.md
-      - name: dotnet nuget enable source
-        href: ../core/tools/dotnet-nuget-enable-source.md
-      - name: dotnet nuget list source
-        href: ../core/tools/dotnet-nuget-list-source.md
-      - name: dotnet nuget remove source
-        href: ../core/tools/dotnet-nuget-remove-source.md
-      - name: dotnet nuget update source
-        href: ../core/tools/dotnet-nuget-update-source.md
-      - name: dotnet nuget verify
-        href: ../core/tools/dotnet-nuget-verify.md
-      - name: dotnet nuget trust
-        href: ../core/tools/dotnet-nuget-trust.md
-      - name: dotnet nuget sign
-        href: ../core/tools/dotnet-nuget-sign.md
-    - name: dotnet pack
-      href: ../core/tools/dotnet-pack.md
-    - name: dotnet publish
-      href: ../core/tools/dotnet-publish.md
-    - name: dotnet restore
-      href: ../core/tools/dotnet-restore.md
-    - name: dotnet run
-      href: ../core/tools/dotnet-run.md
-    - name: dotnet sdk check
-      href: ../core/tools/dotnet-sdk-check.md
-    - name: dotnet sln
-      href: ../core/tools/dotnet-sln.md
-    - name: dotnet store
-      href: ../core/tools/dotnet-store.md
-    - name: dotnet test
-      href: ../core/tools/dotnet-test.md
-    - name: dotnet tool
-      items:
-      - name: dotnet tool install
-        href: ../core/tools/dotnet-tool-install.md
-      - name: dotnet tool list
-        href: ../core/tools/dotnet-tool-list.md
-      - name: dotnet tool restore
-        href: ../core/tools/dotnet-tool-restore.md
-      - name: dotnet tool run
-        href: ../core/tools/dotnet-tool-run.md
-      - name: dotnet tool search
-        href: ../core/tools/dotnet-tool-search.md
-      - name: dotnet tool uninstall
-        href: ../core/tools/dotnet-tool-uninstall.md
-      - name: dotnet tool update
-        href: ../core/tools/dotnet-tool-update.md
-    - name: dotnet vstest
-      href: ../core/tools/dotnet-vstest.md
-    - name: dotnet watch
-      displayName: hot reload
-      href: ../core/tools/dotnet-watch.md
-    - name: dotnet workload
+    - name: dotnet command reference
```

Comment:
I see the logic behind grouping these nodes under a "command reference" node, but the nodes under this new node are already recognizable without the new node in that all of them have names that begin with `dotnet`. And doing this adds an extra level of nesting in a TOC that is already two levels deep at that point. Have you seen any data indicating that an extra node is needed for clarity?

Reply:
We used to have a Reference node like this but removed it to flatten the TOC, see https://github.com/dotnet/docs/pull/25222

Reply:
We discussed this in the sprint meeting this morning and agreed that the long list of `dotnet xyz` entries obscures the other nodes under CLI, such as tab completion and templates. My thought is that the extra level is worth it to have them all organized together. Also, in the proposed breakout of TOCs, tools and diagnostics would be a separate TOC and this would likely go back to only 3 levels deep. cc @adegeo